app/components/Header/index.tsx

```diff
@@ -18,7 +18,7 @@ const StyledHeader = styled(Layout.Header)`
       height: ${(props) => props.theme.headerHeight};
       align-items: center;
       justify-content: center;
-      background-color: ${colors.primary};
+      background-color: ${colors.lightGreen};
```

Comment:
Why have changed this ? Any reason ?

Reply:
Tried to make UI more appealing and hence made those changes. I have added screenshots in the PR description for the reference.